src/octopeer-github/content/ContentController.ts

```diff
@@ -189,6 +190,19 @@ class ContentController {
     }
 
     /**
+     * Custom JQuery event finishScroll. Gets triggered when there hasn't been a scroll event for 500ms.
+     */
+    private finishScrollEvent() {
```

Comment:
The naming looks like it actually fires a `scroll:finish` event. Maybe call it something like `setUpFinishScrollEvent`?